iiab

```diff
@@ -429,6 +429,11 @@ fi
 
 touch $FLAGDIR/iiab-complete
 
+if ! [[ $(ls -la /sys/class/net/*/phy80211 | awk -F / '{print $5}' | wc ) == 0 ]]; then
```

Comment:
Does the suggestion (further below) also work?

If so, it suppresses this error on non-WiFi systems:

```
root@box:~# ls -la /sys/class/net/*/phy80211 | awk -F / '{print $5}' | wc
ls: cannot access '/sys/class/net/*/phy80211': No such file or directory
      0       0       0
```
```
root@box:~# ls -la /sys/class/net/*/phy80211
ls: cannot access '/sys/class/net/*/phy80211': No such file or directory
```

FWIW: `2>/dev/null` would also work below, if preserving STDOUT is desired:

```suggestion
if ls /sys/class/net/*/phy80211 &>/dev/null; then
```

Reply:
might be useful to know why iiab-network is not being run, lack of phy80211 in any of the interfaces...

Reply:
Currently this PR does not display STDOUT e.g. network devices like ap0 and wlan0.

If you want to display both STDOUT and STDERR, here's one rudimentary (bare bones) option:

```
if ls -d1 /sys/class/net/*/phy80211; then
```

When run, it shows stuff like...

```
root@box:~# if ls -d1 /sys/class/net/*/phy80211; then :; fi
/sys/class/net/ap0/phy80211
/sys/class/net/wlan0/phy80211
```

```
root@box:~# if ls -d1 /sys/class/net/*/phy80211; then :; fi
ls: cannot access '/sys/class/net/*/phy80211': No such file or directory
```

Reply:
Just a note that this PR's original if statement always returns True.
That aspect can be fixed with `wc -l` instead of `wc`.

However that's not a comprehensive fix, as it still does not display STDOUT.

Reply:
I don't really care what the syntax you decide to push is just make it happen so users that use the 'iiab' script outside of the intended target of supporting offline wifi serving of web content can continue to muck around without bothering to read any of the docs, reducing frivolous requests for support of their local network layout.